js/core/utils/position.js

```diff
@@ -1,9 +1,41 @@
+import $ from '../renderer';
 import config from '../config';
+import typeUtils from '../utils/type';
 
 const getDefaultAlignment = function(isRtlEnabled) {
     const rtlEnabled = isRtlEnabled ?? config().rtlEnabled;
 
     return rtlEnabled ? 'right' : 'left';
 };
 
+const getBoundingRect = (instance) => {
+    const element = $(instance).get(0);
```

Comment:
I don't think it's a good idea to use renderer in the utility. It enlarges coupling. I'm sure that utility that works with DOM nodes do not need to know about the `renderer` entity. Maybe only DOM nodes should be allowed as an entry argument for this method?

Reply:
I thought about it. I agree that coupling  enlarges, but also util external api becomes more flexible. I decided that the second aspect is more important